### DISCUSSION TITLE
Local Dev Set up for sherpa-onnx/csrc

### DISCUSSION THREAD

**Discussion by iprovalo** [k2-fsa/sherpa-onnx#1115] [General]
I could not find any documentation for setting up the local dev env for the sherpa-onnx/csrc.  Is there a recommended set up for local development IDE?

Thank you!

**Comment by csukuangfj** [comment]
sherpa-onnx is a CMake-based project.

We are using `vim` with a terminal for development.

You can use any IDE you want.

**Reply by iprovalo** [reply]
Thank you!
